i18n/ja.toml

```diff
@@ -0,0 +1,44 @@
+[languages]
+other = "言語"
+
+[gdquest_title]
+other = "ゲーム開発者になろう"
+
+[gdquest_subtitle]
+other = "無料のオープンソースソフトウェアでプロ級のゲームを作ろう"
```

Comment:
@NathanLovato I translated "professional games" to "games looks like it was developed by professional game developers" in Japanese.
How do you think about this translation?

Reply:
It's great, thanks. In general, I'm all for translations that best convey the meaning or the idea in the culture rather than translations that try to stay as close as possible to the original phrasing. Feel free to adapt the wording to what feels most natural or clearest in Japanese.

Reply:
Thank you for your quick reply.
It's a good policy!!
I think other sentences were also translated without significantly changing the meaning.
("Getting started" was very difficult to translate natural expression in Japanese😅)